### PR TITLE
project into the next week

### PR DESCRIPTION
**Pull request by neon-ninja**
This PR adds another trace, extrapolating the current change in slope into the next week. This new trace only shows when the animation is complete - ie, on the latest day. Live demo here: https://raw.githack.com/UoA-eResearch/covidtrends/projections/

### CODE REVIEW

**Comment by aatishb** [comment]
Thanks for sharing, this is definitely an interesting idea. I'm nervous about adding projections / predictions as there might be a high cost to making a wrong call, so we want to tread carefully. For now I'm curious to hear a bit more on how you're doing this, it seems like some kind of extrapolation of the past 7 days? 

One potential issue is it seems to trip up if there's a day where no numbers are reported (which happens occasionally). e.g. 
Covid Trends(3)

Thinking aloud: perhaps a range/confidence interval in addition to a trend line would be more indicative of the uncertainty? 

Not asking you to add these right now, I'm just curious to hear more about your approach, and would like to think about whether this makes sense for this branch.

**Comment by rpkoller** [comment]
I would also be extremely cautious about projections and predictions at the moment. Those might be instrumentalized one way or another. The german Robert Koch Institute published a paper about estimates of the epidemiological development. It contains  a graph of the expected development of the effective reproduction number R.

Bildschirmfoto 2020-04-19 um 16 42 47
https://www.rki.de/DE/Content/Infekt/EpidBull/Archiv/2020/Ausgaben/17_20_SARS-CoV2_vorab.pdf

Point is the estimate of R went beneath 1 a few days before the 23rd of March where the lockdown and social distancing measures started in Germany. The value kept around 1 the following days and weeks. The conclusions of people voting to open up the country again early on is that the lockdown had actually no effect at all and that the maximum of the epidemic was already reached before and we are already in the aftermath. But if you take in contrast a look at the graph of Germany in Covidtrends the last time of exponential growth was around the 2nd or 3rd of April (so the measurements actually showed an effect).
Basically I like the idea of some sort of projection but in the current situation it might be problematic and counter productive. It could be simply instrumentalized due to the aforementioned reasons and be misread. :/

**Comment by aatishb** [comment]
I agree that we don't want to get into prediction on this branch. We're unlikely to get it right, and the cost of spreading harmful information is high. Also (as we're learning) [predictions that extrapolate using past data are fairly unreliable](https://www.sydney.edu.au/content/dam/corporate/documents/centre-for-translational-data-sience/statistical_accuracy_covid19_predictions_ihme_model.pdf) (compared to epidemiological model based predictions). I thought [this gif](https://twitter.com/clcrozier/status/1251148890595708938) was a neat illustration of the risks of extrapolating forward.
